wagtail/admin/views/pages/edit.py

```diff
@@ -867,6 +944,12 @@ def get_context_data(self, **kwargs):
             instance=self.page, request=self.request, form=self.form
         )
         action_menu = PageActionMenu(self.request, view="edit", page=self.page)
+        side_panels = [
+            StatusSidePanel(self.page_for_status),
+            HistorySidePanel(self.page_for_status),
+            CommentsSidePanel(self.page_for_status),
+            PreviewSidePanel(self.page_for_status),
```

Comment:
I imagine for now we’d need to remove this:

```suggestion
            # PreviewSidePanel(self.page_for_status),
```